docs/interpreter/spark.md

```diff
@@ -40,6 +40,73 @@ Spark Interpreter group, which consisted of 4 interpreters.
 </table>
 
 ## Configuration
+Zeppelin provides the below properties for Spark interpreter. 
```

Comment:
What do you think mentioning that user can add any spark properties even though they're not listed by Zeppelin?